packages/v3/src/components/CvSkeletonText/CvSkeletonText.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <p
+    v-if="lineCount === 1"
+    :class="[
+      `${carbonPrefix}--skeleton__text`,
+      { [`${carbonPrefix}--skeleton__heading`]: heading },
+    ]"
+    :style="{ width }"
+  />
+  <div v-else>
+    <p
+      v-for="i in lineCount"
+      :key="i"
+      :class="[
+        `${carbonPrefix}--skeleton__text`,
+        { [`${carbonPrefix}--skeleton__heading`]: heading },
+      ]"
+      :style="{ width: widths[i - 1] }"
+    />
+  </div>
+</template>
+
+<script>
+import { computed } from 'vue';
+import { carbonPrefix } from '../../global/settings';
+
+const LENGTH_MODIFIERS = [`64px`, `2px`, `32px`];
```

Comment:
Is there a good reason to use these values instead of the Carbon react method?

```
const randoms = [0.973051493507435, 0.15334737213558558, 0.5671034553053769];

function getRandomInt(min, max, n) {
  return Math.floor(randoms[n % 3] * (max - min + 1)) + min;
}
```

Reply:
I chose to implement it more naively because the Carbon React method kind of confused me at first. My implementation replicates the behavior while being more straight-forward, but there's no reason not to go with the React implementation :)

Reply:
Fair enough